README.md

```diff
@@ -1 +1,29 @@
-# Serializer
\ No newline at end of file
+# WIP: Cerializer
+Cerializer aims at providing a even faster alternative to FastAvro and Avro standard library.
+
+This speed increase does not come without a cost. Cerializer will work only with predefined set of schemata for which it will generate tailor made code. This way, the overhead caused by the universality of other serialization libraries will be avoided.
+
+Special credit needs to be given to [FastAvro](https://github.com/fastavro/fastavro) library, by which is this project heavily inspired.
+
+**Usage Example:**
+1. Put all your schemata into a schemata root folder following this pattern.<br>
+`[schemata_root_folder]/[namespace]/[schema_name]/[schema_version]/schema.yaml`<br>
+It is essential that you follow this pattern and safe your schemata in `schema.yaml` files.
```

Comment:
Is there a way to use it easily without this structure and yaml files? It is only quantlane specific so if you want to offer it to the public you should have interface that will e.g. work with schema passed as a dict.